src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputModelPropertyConverter.cs

```diff
@@ -32,22 +32,22 @@ private static InputModelProperty ReadInputModelProperty(ref Utf8JsonReader read
             InputType? propertyType = null;
             InputConstant? defaultValue = null;
             bool isReadOnly = false;
-            bool isRequired = false;
+            bool isOptional = false;
```

Comment:
Should we include the nullable change in InputModelProperty in this repo as well? and seems we should get the consolidation of input types done sooner to avoid the duplicated work

Reply:
what is the nullable change?

Reply:
leave it as it is for now, when we implement all the features in Azure Plugin, deserialization of InputType will be removed from autorest.csharp.